### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by clcheatle**

### CODE REVIEW

**Review by nicholasvvong — APPROVED** [review]
Added follow functionality. Removed followingmovies related things from this service (meant to be in movie service)

File: Logic/UserLogic.cs [hunk 1]
```diff
@@ -70,29 +80,33 @@ public async Task<bool> UpdatePermissions(string userid, int permissionLevel)
             return await _repo.UpdatePermissions(userid, permissionLevel);
         }
 
-        public async Task<List<string>> GetFollowingMovies(string userid)
+        public async Task<bool> FollowUser(string followerid, string followeeid)
+        {
+            Repository.Models.FollowingUser newFollow = new Repository.Models.FollowingUser();
+            newFollow.FollowerUserId = followerid;
+            newFollow.FolloweeUserId = followeeid;
```
Comment (markmoorerev) [inline Logic/UserLogic.cs]: duplicate?

**Review by markmoorerev — CHANGES_REQUESTED** [review]
correct that one duplicate line.

File: Logic/UserLogic.cs (hunk 1 above)
Comment (nicholasvvong) [reply Logic/UserLogic.cs]: one is follower, one is followee

**Review by markmoorerev — APPROVED** [review]

**Pull request merged**